### ISSUE TITLE
Will there be support for cct.

### ISSUE THREAD

**Issue by VadOs1** [locationtech/proj4j#114]
Hi,

I'd like to be able to execute cct application of proj using proj4j. 

https://proj.org/en/9.5/apps/cct.html#cct

Do you guys support it or have any plans to add it?

`echo 17.7562015132 45.3935192042 133.12 2017.8 | cct +proj=cart`
https://proj.org/en/9.5/operations/conversions/cart.html

Thank you

**Comment by dr-jts** [comment]
Seems like a great addition to the project.  CLI tools are always nice to have.

**Comment by pomadchin** [comment]
Agree, that's a nice feature to have. If you decide at some point to implement it I'd be happy to help!
